src/utils.ts

```diff
@@ -1407,3 +1407,23 @@ export function commandSupportsReadConcern(command: Document, options?: Document
 
   return false;
 }
+
+/**
+ * A utility function to get the instance of mongodb-client-encryption, if it exists.
+ *
+ * @throws MongoMissingDependencyError if mongodb-client-encryption isn't installed.
+ * @returns
+ */
+export function getMongoDBClientEncryption() {
+  let mongodbClientEncryption;
+
+  // Note (NODE-4254): This is to get around the circular dependency between
```

Comment:
```suggestion
  // NOTE(NODE-4254): This is to get around the circular dependency between
```
Just making the formatting consistent